### PR TITLE
fix: Point to v4 compatible branch

### PR DESCRIPTION
**Pull request by Uzaaft**

### CODE REVIEW

File: lua/config/lazy.lua [hunk 1]
```diff
@@ -18,7 +18,8 @@ if USE_STABLE then table.insert(spec, { import = "astronvim.lazy_snapshot" }) en
 require("lazy").setup {
   spec = vim.list_extend(spec, {
     -- AstroCommunity import any community modules here
-    -- { "AstroNvim/astrocommunity" },
+    -- TODO: Remove branch v4-refactor on release
+    { "AstroNvim/astrocommunity", branch = "v4-refactor" },
```
Comment (mehalter) [inline lua/config/lazy.lua]:
```suggestion
    { "AstroNvim/astrocommunity", branch = "v4" },
```

This should just be v4 to align the branch names with all of the other repos

**Review by mehalter — CHANGES_REQUESTED** [review]

File: lua/config/lazy.lua [hunk 2]
```diff
@@ -18,7 +18,8 @@ if USE_STABLE then table.insert(spec, { import = "astronvim.lazy_snapshot" }) en
 require("lazy").setup {
   spec = vim.list_extend(spec, {
     -- AstroCommunity import any community modules here
-    -- { "AstroNvim/astrocommunity" },
+    -- TODO: Remove branch v4-refactor on release
+    { "AstroNvim/astrocommunity", branch = "v4" },
```
Comment (mehalter) [inline lua/config/lazy.lua]:
```suggestion
    -- { "AstroNvim/astrocommunity", branch = "v4" },
```

**Review by mehalter — CHANGES_REQUESTED** [review]

File: lua/config/lazy.lua [hunk 3]
```diff
@@ -18,7 +18,8 @@ if USE_STABLE then table.insert(spec, { import = "astronvim.lazy_snapshot" }) en
 require("lazy").setup {
   spec = vim.list_extend(spec, {
     -- AstroCommunity import any community modules here
-    -- { "AstroNvim/astrocommunity" },
+    -- TODO: Remove branch v4-refactor on release
```
Comment (Uzaaft) [inline lua/config/lazy.lua]:
```suggestion
    -- TODO: Remove branch v4 on release
```

**Review by mehalter — APPROVED** [review]
LGTM

**Pull request merged**